src/components/Map.tsx

```diff
@@ -19,9 +21,13 @@ function Map() {
       /**
        * Initialize application
        */
+
+      const globalApikey =
+        "AAPK66d0fa7f737142fcb780f9fd1c8155f04YBLPof_SmYB-dzJgafylrodNieETG6SVXdYeo75C5eFIt9NykPed6jRtTI_c01w";
+      esriConfig.apiKey = globalApikey;
```

Comment:
Do we want to use an env var for the API key?

Reply:
We can! I was just playing around with it to make sure it worked and figured I could change it later. env var would probably be better. The only thing is we are probably going to have multiple API keys since we should have fine grained ones where we can. Can't have one for webmaps so had to do global.

Reply:
Idk that seems weird honestly. Usually you have one API Key for an app and just give it as much priv as needed. And then hide it in a env var.

Reply:
@benelan - I'll do a new PR for env vars in case I break something!

Reply:
Cool, switch the key when you do cuz this one is leaked.